### DISCUSSION TITLE
No Running Event Loop

### DISCUSSION THREAD

**Discussion by fabthegreat** [django/channels#1810] [Q&A]
Hi,

my request has already been raised to Stackoverflow but still I have been advised to put my question here as well:
https://stackoverflow.com/questions/71145039/django-channels-and-running-event-loop

I have tried several asgiref versions, but I am questionning my self if I have the right to call get_event_loop in the consumer itself...
Is that still a bug or a misuse?

Thanks

**Comment by mikbuch** [comment]
Hi, if you are interested in some alternative solutions, see my answer on the Stack: https://stackoverflow.com/a/71294371/8877692

Sorry I cannot help you directly with your question, but I did not yet try running event loops myself. I usually have sender-receiver sync use-cases in channels, so I didn't need multiprocessing in Django-channels yet.
